definitions/tools/filter_vcf_somatic_llr.cwl

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "use the binomial/llr somatic filter to weed out low confidence variants"
+requirements:
+    - class: DockerRequirement
+      dockerPull: mgibio/somatic-llr-filter
+baseCommand: ["/opt/conda/bin/python3","/usr/bin/somatic_llr_filter.py"]
+arguments: 
+    ["--overwrite", #we expect to have to overwrite the SOMATIC field
+    {valueFrom: "$(runtime.outdir)/somatic_llr_filtered.vcf"}]
+inputs:
+    vcf:
+        type: File
+        inputBinding:
+            position: -1
+    threshold: 
+         type: float
+         inputBinding:
+             prefix: "--llr-threshold"
+             position: -2
+outputs:
+     somatic_llr_filtered_vcf:
+         type: File
+         outputBinding:
+             glob: "somatic_llr_filtered.vcf"
```

Comment:
The formerly final filter yielded a generic name:
https://github.com/genome/analysis-workflows/blob/44ada20f3eeb59005d5bd999d2435102e9bae991/definitions/tools/filter_vcf_cle.cwl#L23

I don't have great suggestions on what to do for this one, but it seems like we may want to take a pass at all our naming through variant detection and filtration.

Reply:
Yeah, I'm going to punt on it for this PR, and make an issue suggesting that we do some name harmonization